.github/renovate.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": [
+        "config:js-app"
```

Comment:
Does this mean we are stating it a javascript app or `js-app` means something else?
couldn't find details here: https://docs.renovatebot.com/configuration-options/#extends

Reply:
Their documentation is not very well organized. :slightly_smiling_face: You need to look here: https://docs.renovatebot.com/presets-config/#configjs-app

It's a preset that should give us some reasonable base behavior for JS apps (compared to JS libraries where dependency pinning is undesirable etc.).

Reply:
ack, thanks for the knowledge share.